src/parsingCopy/src/com/company/ParseTest.java

```diff
@@ -0,0 +1,107 @@
+package com.company;
+
+import org.eclipse.cdt.core.dom.ast.IASTNode;
+import org.eclipse.cdt.internal.core.dom.parser.cpp.CPPASTLiteralExpression;
+import org.eclipse.cdt.internal.core.dom.parser.cpp.CPPASTName;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ParseTest {
+    @Test
+    public void checkLitExpr() throws Exception {
+        String code = "const uint16_t CAN_ID = 0x16; \n" +
+                "const uint16_t CAN_ID2 = 0x32; \n" +
+                "const uint16_t CAN_ID3 = 0x64;";
+
+        Parse test = new Parse(code);
+
+        String value1 = "0x16";
+        String value2 = "0x32";
+        String value3 = "0x64";
+
+        String n1 = "CAN_ID";
+        String n2 = "CAN_ID2";
+        String n3 = "CAN_ID3";
+
+        CPPASTName name1 = new CPPASTName(n1.toCharArray());
+        CPPASTName name2 = new CPPASTName(n2.toCharArray());
+        CPPASTName name3 = new CPPASTName(n3.toCharArray());
+
+        String litExpr1 = test.get(name1).getRawSignature();
```

Comment:
Would it make sense to have `get` take a string as its argument instead of this `CPPASTName`?